Product/Production/Common/Properties/src/main/resources/gateway.properties

```diff
@@ -109,6 +109,12 @@ CoreX12GenericBatchTimeStampTimeToLive=300
 # This is only valid for  Core X12 Batch request and response services.
 CoreX12GenericBatchFutureTimeToLive=60
 
+# enableAuthDEvidenceConditionsDefaultValue accepts true or false, if not provided defaults to true.
```

Comment:
Example: "Boolean for adding the default values for AuthDecision Before and After Conditions.  Default property value is false."